Sources/DangerSwiftPeripherySampleApp/DangerSwiftPeripherySampleApp.swift

```diff
@@ -2,6 +2,8 @@ public struct DangerSwiftPeripherySampleApp {
     public private(set) var text = "Hello, World!"
 
     public init() {
```

Comment:
<!--
  0 failure: 
  1 warning:  Initializer 'init...
  
  
  DangerID: danger-id-Danger;
  File: Sources/DangerSwiftPeripherySampleApp/DangerSwiftPeripherySampleApp.swift;
  Line: 4;
-->

- :warning: Initializer 'init()' is declared public, but not used outside of DangerSwiftPeripherySampleApp